src/components/pages/TeachersPage/exampleChats.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * Not meant to be used in production.
+ * Replace studentChats in ./index.tsx to test out UI with multiple chats during development
+ */
+
+export const EXAMPLE_CHATS = [
```

Comment:
This should be a `.ts` file.  Then you can put `import { StudentChat } from '.';` at the top of this file and export the StudentChat interface form ./`index.tsx
By making it a typescript file it ensures that if anyone changes the StudentChat interface they'll get a warning to change this Example Chat also.

```suggestion
export const EXAMPLE_CHATS: StudentChat[] = [
```